docs/pages/docs/usage/configuration.mdx

```diff
@@ -95,29 +90,37 @@ export default getRequestConfig(async ({locale}) => {
 });
 ```
 
-To read configured messages in a component, you can use the `useMessages` hook:
+After messages are configured, they can be used via `useTranslations`.
+
+In case you require access to messages in a component, you can use a convenience API to read them from your configuration:
 
 ```tsx
+// Regular components
 import {useMessages} from 'next-intl';
-
 const messages = useMessages();
+
+// Async Server Components
+import {getMessages} from 'next-intl/server';
+const messages = await getMessages();
 ```
 
 </Tab>
 <Tab>
 
 ```tsx
-import {NextIntlClientProvider, useMessages} from 'next-intl';
+import {NextIntlClientProvider} from 'next-intl';
+import {getMessages} from 'next-intl/server';
 
-// Read messages configured via `i18n.ts`. Alternatively,
-// messages can be fetched from any other source too.
```

Comment:
Reading from other sources shouldn't be relevant here (see also "How can I load messages from remote sources?" above).